assets/stylesheets/purple/main.scss

```diff
@@ -20,7 +20,7 @@ body {
   padding-top: 60px;
   .content {
     margin-bottom: 90px;
-    > a {
+    p, a {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own single line